subgraphs/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+services:
+  graph-node:
+    image: graphprotocol/graph-node
+    ports:
+      - '8000:8000'
+      - '8001:8001'
+      - '8020:8020'
+      - '8030:8030'
+      - '8040:8040'
```

Comment:
Take a look at the changes proposed in the following PR for improvement opportunities here:

https://github.com/graphprotocol/graph-node/pull/5763/files

Reply:
I will apply them

Reply:
Applied here [14a86b8](https://github.com/hemilabs/ui-monorepo/pull/805/commits/14a86b84dec64532489d9f0fa0c628e664afd452)